ngraph/src/ngraph/op/multiply.hpp

```diff
@@ -58,8 +58,8 @@ namespace ngraph
             class NGRAPH_API Multiply : public util::BinaryElementwiseArithmetic
             {
             public:
-                static constexpr NodeTypeInfo type_info{"Multiply", 1};
-                const NodeTypeInfo& get_type_info() const override { return type_info; }
+                NGRAPH_RTTI_DECLARATION("Multiply", Node, 1);
```

Comment:
Why you don not set `BinaryElementwiseArithmetic` as a base class?

Reply:
It is likely a correct decision. But it is not used right now. So I avoided to introduce extra levels of hierarchy without clear need to do that.